spec.html

```diff
@@ -948,7 +948,7 @@ <h1>The Boolean Type</h1>
-      <p>Where ECMAScript operations interpret String values, each element is interpreted as a single UTF-16 code unit. However, ECMAScript does not place any restrictions or requirements on the sequence of code units in a String value, so they may be ill-formed when interpreted as UTF-16 code unit sequences. Operations that do not interpret String contents treat them as sequences of undifferentiated 16-bit unsigned integers. The function `String.prototype.normalize` (see <emu-xref href="#sec-string.prototype.normalize"></emu-xref>) can be used to explicitly normalize a String value. `String.prototype.localeCompare` (see <emu-xref href="#sec-string.prototype.localecompare"></emu-xref>) internally normalizes String values, but no other operations implicitly normalize the strings upon which they operate. Only operations that are explicitly specified to be language or locale sensitive produce language-sensitive results.</p>
+      <p>Where ECMAScript operations interpret String values, each element is interpreted as a single UTF-16 code unit. As defined in section 3.8 of the Unicode Standard (Surrogates), a <dfn id="high-surrogate-code-unit">high-surrogate code unit</dfn> (or <dfn id="leading-surrogate">leading surrogate</dfn>) is a code unit with a numeric value in the inclusive range 0xD800 to 0xDBFF and a <dfn id="low-surrogate-code-unit">low-surrogate code unit</dfn> (or <dfn id="trailing-surrogate">trailing surrogate</dfn>) is a code unit with a numeric value in the inclusive range 0xDC00 to 0xDFFF. However, ECMAScript does not place any restrictions or requirements on the sequence of code units in a String value, so they may be ill-formed when interpreted as UTF-16 code unit sequences. Operations that do not interpret String contents treat them as sequences of undifferentiated 16-bit unsigned integers. The function `String.prototype.normalize` (see <emu-xref href="#sec-string.prototype.normalize"></emu-xref>) can be used to explicitly normalize a String value. `String.prototype.localeCompare` (see <emu-xref href="#sec-string.prototype.localecompare"></emu-xref>) internally normalizes String values, but no other operations implicitly normalize the strings upon which they operate. Only operations that are explicitly specified to be language or locale sensitive produce language-sensitive results.</p>
```

Comment:
While we're changing this anyway, I'd suggest starting a new paragraph before:

> The function `String.prototype.normalize` [...]

Currently, it reads as if normalization has any kind of effect on surrogates (but it doesn't).

Reply:
Good point. I reorganized a bit more.